README.md

```diff
@@ -21,4 +21,44 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
 ### Quick Start
 
-Go [here](docs/README.md) for an example of how to get up and going with the cluster api using vSphere.
\ No newline at end of file
+Go [here](docs/README.md) for an example of how to get up and going with the cluster api using vSphere.
+
+### Where to get the vSphere provider container
```

Comment:
i think we also need to doc CAPI manager container image, that should be versioned as v1alpha1

Reply:
I don't think it's going to be tagged as v1alpha1.  The apis is labeled v1alpha1.  I'm not sure what the version for CAPI will be, and when that release is going to happen, and who is pushing it.

Reply:
@vincepri @detiber i remember this been discussed on CAPI meeting, we want to move away from relying on `latest` tag of CAPI manager image.  IMO, CAPI should push the image with tag when release, isn't this the plan?

Reply:
looks like it going to release version `0.1.0`, `gcr.io/k8s-cluster-api/cluster-api-controller:0.1.0`
based on PR https://github.com/kubernetes-sigs/cluster-api/pull/861

Reply:
Yes, down with `latest` :joy:

Reply:
Well that's unfortunate.  I looked at kubeadm and they are using `v` in front of release versions.

Reply:
Ideally, it should be uniform across all sig-cluster-lifecycle projects.